### PR TITLE
Create issue templates and instructions for release testing

### PR DESCRIPTION
**Pull request by jashapiro**
I used github's features to create some issue templates.

Originally, I was just going to create the create an issue template for new releases, but it seemed easy enough to just make all the templates at once. 

I used the provided templates for features and bugs, with some modifications, especially for the bug report one.
I expect features to be the one we use most of the time (or no template), and it is the most general.

The release checklist is the one that should need the most review though:
- Did I describe the steps to test a release candidate sufficiently? 
- Should I include more detail about what happens after all the tests pass? (Instructions for actually creating a release?)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (jaclyn-taroni) [inline .github/ISSUE_TEMPLATE/bug_report.md:2]: This looks good 👍🏻

File: .github/ISSUE_TEMPLATE/feature_request.md
Comment (jaclyn-taroni) [inline .github/ISSUE_TEMPLATE/feature_request.md:2]: This one looks good, too 👍🏻

File: .github/ISSUE_TEMPLATE/release-checklist.md
Comment (jaclyn-taroni) [inline .github/ISSUE_TEMPLATE/release-checklist.md:12]: Do you mean mark this issue (the release) as blocked by the unresolved issues in ZenHub?

File: .github/ISSUE_TEMPLATE/release-checklist.md
Comment (jaclyn-taroni) [inline .github/ISSUE_TEMPLATE/release-checklist.md:20]: I believe your point here is: You might actually want to run something on some data. Is that correct?

**Review by jaclyn-taroni — COMMENTED** [review]
This looks pretty good! I had a couple questions about the release checklist, which I believe can be resolved with rewording things. 

Regarding this question:

> Should I include more detail about what happens after all the tests pass? (Instructions for actually creating a release?)

My inclination is yes. I don't expect that it will need to be lengthy. I think you could change the current section title to make it more explicitly about testing/preparing for a release and add a new section `Creating a release`

File: .github/ISSUE_TEMPLATE/release-checklist.md
Comment (jashapiro) [reply .github/ISSUE_TEMPLATE/release-checklist.md:12]: Yes, I will reword.

File: .github/ISSUE_TEMPLATE/release-checklist.md
Comment (jashapiro) [reply .github/ISSUE_TEMPLATE/release-checklist.md:20]: Sort of... There are default libraries in the workflow, but if there is a particular system to test, others might be good choices. I will reword.

**Comment by jashapiro** [comment]
I updated some wording and added release instructions. The package was failing tests yesterday because it seems to be failing to download and install some dependencies. Not fully sure what to do about that, and it usually seems to be transient. If it fails again I might look into it a bit more in the interests of avoiding similar problems in the future.

File: .github/ISSUE_TEMPLATE/release-checklist.md
Comment (jaclyn-taroni) [inline .github/ISSUE_TEMPLATE/release-checklist.md:15]:
```suggestion
- [ ] Did all automated tests and the Docker build in this repo pass in CI? (You may have to wait for the docker image to build. Check [Github actions](https://github.com/AlexsLemonade/scpcaTools/actions/workflows/build-docker.yaml) before proceeding to the next step)
```

Or maybe the `)` goes after "wait for the docker image to build."

**Review by jaclyn-taroni — APPROVED** [review]
Caught one missing ), but other than that LGTM!

File: .github/ISSUE_TEMPLATE/release-checklist.md
Comment (jashapiro) [reply .github/ISSUE_TEMPLATE/release-checklist.md]: Just made it not a parenthetical.

**Comment by jashapiro** [comment]
Now that the tests are passing, I will merge this.

**Pull request merged**
